src/simpla.js

```diff
@@ -148,6 +165,10 @@ const Simpla = new class Simpla {
     return storeToObserver(this._store).observe(pathInStore, wrappedCallback);
   }
 
+  prefetch(path) {
+    return this.find({ ancestor: path });
```

Comment:
Prefetch should just return an empty promise that resolves when data is hydrated, not the data itself